qemu/tests/cfg/migrate.cfg

```diff
@@ -11,7 +11,8 @@
     mig_timeout = 3600
     ping_pong = 1
     pre_command = "sync && echo 3 > /proc/sys/vm/drop_caches"
-    cmds_installed_host = "ksmtuned"
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
```

Comment:
`!Host_RHEL.m6, ****`

Reply:
> `!Host_RHEL.m6, ****:`
Thanks for your help,I have changed it. Now these can test on both rhel.8 and rhel.9. Please help me review it again, thanks a lot.